tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/APIViewService.cs

```diff
@@ -0,0 +1,49 @@
+using Azure.Sdk.Tools.Cli.Services.APIView;
+
+namespace Azure.Sdk.Tools.Cli.Services;
+
+public interface IAPIViewService
+{
+    Task<string?> GetRevisionContent(string apiRevisionId, string reviewId, string contentReturnType, string environment = "production");
+    Task<string?> GetCommentsByRevisionAsync(string revisionId, string environment = "production");
+}
+
+public class APIViewService : IAPIViewService
+{
+    private readonly IAPIViewHttpService _httpService;
+    private readonly ILogger<APIViewService> _logger;
+
+    public APIViewService(
+        IAPIViewHttpService httpService,
+        ILogger<APIViewService> logger)
+    {
+        _httpService = httpService;
+        _logger = logger;
+    }
+
+    public async Task<string?> GetCommentsByRevisionAsync(string revisionId, string environment = "production")
```

Comment:
nit: Using an enum is probably better for environment so agent or cli invoker won't sent some random value.